### ISSUE TITLE
sendTypingEvent triggers onTypingEvent

### ISSUE THREAD

**Issue by alecam** [microsoft/omnichannel-chat-sdk#135]
Hi @xTEddie, it seems that when using latest version (1.1.1-main.6b72ca1), when calling sendTypingEvent(), onTypingEvent gets triggered.

I also tried with 1.1.0 and got the same issue.

Thank you!

**Comment by xTEddie** [comment]
Root cause is `ChatSDK.onTypingEvent()` is not filtering out the current user's sending typing events.

A fix would be to add a filter in the event.

**Comment by xTEddie** [comment]
Currently, it's possible to manually add an additional filter as follow: 

```js

ChatSDK.onTypingEvent((event) => {
  const {sender, recipient} = event;

  if (sender.communicationUserId !== recipient.communicationUserId) {
    // Agent is typing
  }
}); 
```

**Comment by xTEddie** [comment]
Issue should be fixed as of [v1.2.1-main.6149875](https://www.npmjs.com/package/@microsoft/omnichannel-chat-sdk/v/1.2.1-main.6149875)
